converter/src/test/java/gov/cms/qpp/acceptance/ElementLocationTest.java

```diff
@@ -0,0 +1,53 @@
+package gov.cms.qpp.acceptance;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.function.BiPredicate;
+
+import org.jdom2.Element;
+import org.jdom2.located.Located;
+import org.jdom2.xpath.XPathExpression;
+import org.jdom2.xpath.XPathFactory;
+import org.junit.jupiter.api.Test;
+
+import com.google.common.truth.Truth;
+
+import gov.cms.qpp.conversion.Context;
+import gov.cms.qpp.conversion.decode.QrdaDecoderEngine;
+import gov.cms.qpp.conversion.model.Node;
+import gov.cms.qpp.conversion.xml.XmlUtils;
+
+class ElementLocationTest {
+
+	@Test
+	void testGetLineNumber() throws IOException {
+		runTest((child, node) -> child.getLine() == node.getLine());
+	}
+
+	@Test
+	void testGetColumnNumber() throws IOException {
+		runTest((child, node) -> child.getColumn() == node.getColumn());
+	}
+
+	private void runTest(BiPredicate<Located, Node> elementFinder) throws IOException {
+		Path qrdaPath = Paths.get("../qrda-files/valid-QRDA-III-latest.xml");
+		String qrda = new String(Files.readAllBytes(qrdaPath), StandardCharsets.UTF_8);
+
+		Element document = XmlUtils.stringToDom(qrda);
+		Node node = new QrdaDecoderEngine(new Context()).decode(document).getChildNodes().get(2);
+		XPathExpression<?> xpath = XPathFactory.instance().compile(node.getPath());
+		Element element = (Element) xpath.evaluate(document).get(0);
+		Object elementOfNode = element.getChildren()
+			.stream()
+			.map(Located.class::cast)
+			.filter(child -> elementFinder.test(child, node))
```

Comment:
Am I understanding this that all the children `Element`s (aka. `Located`) should have children that have the same line and column number as the one Node (which is the parent)?  That doesn't seem right.  I'm probably misunderstanding something here.

Reply:
We've discussed this now, will be addressed after QPPCT-807